en/upgrades/decimal-upgrade-to-2-03/2-03-changes.rst

```diff
@@ -0,0 +1,185 @@
+	   	-	OrganisationType
+	   	-	OtherIdentifierType
+   *	-	PolicyMarker
+	   	-	PolicyMarkerVocabulary
+	   	-	PublisherType
+   *	-	RegionVocabulary
+	   	-	ResultType
+	   	-	SectorVocabulary
+   *	-	TiedStatus
+	   	-	ResultType
+	   	-	VerificationStatus
+   *	-	PublisherType
+	   	-	PolicyMarkerVocabulary
+	   	-	PolicyMarker
+   *	-	RegionVocabulary
+	   	-	TiedStatus
+     	- - 
+
+New Codelists
+-------------
+
+**Non-Embedded:**
+
+- Added AidTypeVocabulary codelist (`#185 <https://github.com/IATI/IATI-Codelists-NonEmbedded/issues/185>`__) (`Discussion link <https://discuss.iatistandard.org/t/add-vocabularies-to-aid-type-included-2-03/847>`__) 
+- Added BudgetNotProvided codelist (`#184 <https://github.com/IATI/IATI-Codelists-NonEmbedded/issues/184>`__) (`Discussion link <https://discuss.iatistandard.org/t/add-budget-exempt-attribute-and-codelist-included-2-03/845>`__) 
+- Added ResultVocabulary codelist (`#181 <https://github.com/IATI/IATI-Codelists/issues/181>`__) (`Discussion link <https://discuss.iatistandard.org/t/results-vocabulary-attribute-option-included-2-03/879>`__)  
+- Added TagVocabulary codelist (`#178 <https://github.com/IATI/IATI-Codelists-NonEmbedded/issues/178>`__) (`Discussion link <https://discuss.iatistandard.org/t/non-statistical-secondary-sectors-excluded-2-03/849>`__) 
+
+Updated Codelists
+-----------------
+
+**Embedded:**
+
+- TransactionType Codelist (`#112 <https://github.com/IATI/IATI-Codelists/issues/112>`__) (`Discussion link <https://discuss.iatistandard.org/t/transactiontype-codes-included-2-03/852>`__)
+ Inlcudes the added codes: Code 12 'Outgoing Pledge' and Code 13 'Incoming Pledge'.
+	
+**Non-Embedded:**
+
+- OrganisationType Codelist (`#113 <https://github.com/IATI/IATI-Codelists/issues/113>`__) (`Discussion link <https://discuss.iatistandard.org/t/organisation-type-codes-additions-included-2-03/858>`__) 
+ Includes the added codes: Code 11 'Local Government', Code 24 'Partner Country based NGO', Code 71 'Private Sector in Provider Country', Code 71 'Private Sector in Aid Recipient Country', Code 73 'Private Sector in Third Country' and Code 90 'Other'.
+
+- IndicatorMeasure codelist (`#179 <https://github.com/IATI/IATI-Codelists-NonEmbedded/issues/179>`__) (`Discussion link <https://discuss.iatistandard.org/t/results-represent-more-than-quantitative-data-included-2-03/872>`__) 
+ Includes the added codes: Code 3 'Nominal', Code 4 'Ordinal' and Code 5 'Qualitative'.
+ 
+ .. _2_03_ruleset_changes:
+
+Ruleset Changes
+================
+ 
+- Added rule: reference element (`#48 <https://github.com/IATI/IATI-Rulesets/issues/48>`__) (`Discussion link <https://discuss.iatistandard.org/t/results-vocabulary-attribute-option-included-2-03/879>`__) 
+  
+- Added rules (regex matches): ref attributes (`#50 <https://github.com/IATI/IATI-Rulesets/issues/50>`__) (`Discussion link <https://discuss.iatistandard.org/t/migration-of-organisationregistrationagency-codelist-to-org-id-guide-included-2-03/851>`__) 
```

Comment:
Flagging that this change will likely be removed due to issues with representing regex in the JSON ruleset files.

Reply:
7d17a5fdcd75195ee1969e1896ab32a4777c9652